### PR TITLE
Rearranged Models, Delegated JSON serialization

### PR DESCRIPTION
**Pull request by bencdelaney**
- Split models from a file to a package
- Squashed DAOs and API objects into a single layer
- Added a second model base class, for statler-specific model functions (specifically serialization)
- **Blew away old migrations. This will require a full database recreation. If that's not a good idea, let me know**

### CODE REVIEW

**Comment by bencdelaney** [comment]
**EDIT: This is now a second draft. With the possible exception of some sketchy commented out tests, I don't have other changes in mind.**

**Please ask questions if anything is unclear!**

File: statler_api/migrations/0001_initial.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 from __future__ import unicode_literals
```
Comment (bencdelaney) [inline statler_api/migrations/0001_initial.py:2]: This file was actually blown away and regenerated, both automatically.

File: statler_api/tests/test_reviews.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+from django.test import TestCase
+from statler_api.models import Review, Play
+from datetime import datetime
+
+from statler_api.tests.test_plays import PlayTestHelper
+
+
+class ReviewTestCase(TestCase):
```
Comment (bencdelaney) [inline statler_api/tests/test_reviews.py:8]: Real unit tests!

File: statler_api/tests/test_playLists.py [hunk 3]
```diff
@@ -1,53 +1,52 @@
...
-        dao.refresh_from_db()
-
-        # Convert it to an API object
-        apiList = PlayList(dao)
-
-        # Check things.
-        self.assertEquals(apiList.url_title, dao.url_title)
-        self.assertEquals(len(apiList.entries), 2)
-        # TODO: Check more things.
+# class PlayListTestCase(TestCase):
```
Comment (bencdelaney) [inline statler_api/tests/test_playLists.py]: I should probably do something with these before I merge...

File: statler_api/tests/test_plays.py [hunk 4]
```diff
@@ -122,57 +121,9 @@ def testCanRefreshPlaysFromDatabase(self):
...
 
         # This updates the title of all plays in the database. This may be a bad idea, but for now it works.
-        PlayDAO.objects.update(title="New Title")
+        Play.objects.update(title="New Title")
 
         play.refresh_from_db()
         self.assertEqual("New Title", play.title)
 
 
-class PlayTestCase(TestCase):
```
Comment (bencdelaney) [inline statler_api/tests/test_plays.py:131]: These are all conversion tests, which aren't meaningful now that the models have been merged.

**Comment by chaneberg** [comment]
Very nice. views.py is much more elegant now.

**Pull request merged**